python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -29,45 +30,80 @@ class TimeWindow(NamedTuple):
 class TimeWindowPartitionsDefinition(
     PartitionsDefinition[TimeWindow],  # pylint: disable=unsubscriptable-object
     NamedTuple(
-        "_TimeWindowPartitions",
+        "_TimeWindowPartitionsDefinition",
```

Comment:
looks like there are no backcompat concerns here since its not whitelisted for serdes, hooray